spk/tt-rss/src/service-setup.sh

```diff
@@ -77,14 +71,16 @@ postinst ()
     fi
 
     # Fix permissions
-    chown ${USER} ${WEB_DIR}/${PACKAGE}/lock
-    chown ${USER} ${WEB_DIR}/${PACKAGE}/feed-icons
-    chown -R ${USER} ${WEB_DIR}/${PACKAGE}/cache
+    set_syno_permissions "${INSTALL_DIR}/var/logs" "${USER}"
```

Comment:
The logs directory is populated with files which may be written by `root` user (during startup) or by upgrade script (run by the daemon as `${USER}` user).